lib/ssl/test/ssl_pem_cache_SUITE.erl

```diff
@@ -512,7 +512,7 @@ new_root_pem_helper(Config, CleanMode,
     %% ConnectedN - state after establishing Nth connection
     %% Cleaned - state after periodical cleanup
     %% DisconnectedN - state after closing Nth connection
-    ?CT_PAL(">>> IntermediateServerKeyId = ~w", [IntermediateServerKeyId]),
+    ?CT_LOG(">>> IntermediateServerKeyId = ~w", [IntermediateServerKeyId]),
```

Comment:
What about this macro?! Should we perhaps use it all the time if we are making this change?! @u3s @RaimoNiskanen

Reply:
do you mean replacing occurrences generated by 'git grep -E `ct:pal|ct:log|ct:print' ./lib/ssl/test` ?

it is only 184 of them, i can do it if you like. macros are a bit more more verbose than plain CT functions - including timestamp, pid for example.

Reply:
Well yes, it should be a query replace! But only if it is beneficial. I think Raimo introduced the macro and it is not used in the rest of the test suites. So I think we should take this opportunity to evaluate if it has  benefit that we like to have in all test suites, or is the benefit local to the specific test suite?

Reply:
It was actually me who introduced it, because in ssl_test_lib there was a lot of redundant code collecting timestamp and pid which cluttered the module.

Raimo did rename from PAL to CT_PAL, probably because there was some naming conflict.

I will do a commit with query replace in ssl/test ...

Reply:
Ah, ok git blame is not perfect :)

Reply:
Done in added commit